tools/slang-test/test-context.cpp

```diff
@@ -138,24 +174,24 @@ SlangResult TestContext::_createJSONRPCConnection(RefPtr<JSONRPCConnection>& out
 
 void TestContext::destroyRPCConnection()
 {
-    if (m_jsonRpcConnection)
+    if (m_jsonRpcConnections[slangTestThreadIndex])
     {
-        m_jsonRpcConnection->disconnect();
-        m_jsonRpcConnection.setNull();
+        m_jsonRpcConnections[slangTestThreadIndex]->disconnect();
+        m_jsonRpcConnections[slangTestThreadIndex].setNull();
```

Comment:
To be clear, this code here is a sign to me that we are letting these different features get tangled together in ways that are unhealthy.